### DISCUSSION TITLE
Has @AbdBarho abandoned this project?

### DISCUSSION THREAD

**Discussion by cloudaxes** [AbdBarho/stable-diffusion-webui-docker#690] [General]
@SachiaLanlus has had a pull request to update Automatic1111 outstanding for 2 months without a response from @AbdBarho.  [https://github.com/AbdBarho/stable-diffusion-webui-docker/pull/673]

@AbdBarho, if you're still there and interested in continuing this project, we would welcome your opinion on the pull request.

**Comment by DevilaN** [comment]
Seems a bit like so. I've even reviewed this PR and currently I am working on my fork with v1.9.3 too.

If you take a look on that PR it is nothing complicated. Corrected dependencies and changed commit to proper version. It just works :-)

**Comment by cloudaxes** [comment]
Yea, I'm currently running Automatic1111 v1.9.3 off of @SachiaLanlus's branch.  Works without issue.

**Comment by AbdBarho** [comment]
Sorry for being away for so long, I apologise, life has not treated me well and could not find any time to work on this. I will try to get back to you over this weekend.
